driver-core/src/test/unit/com/mongodb/MongoClientSettingsSpecification.groovy

```diff
@@ -50,7 +50,7 @@ class MongoClientSettingsSpecification extends Specification {
         settings.clusterSettings == ClusterSettings.builder().build()
         settings.connectionPoolSettings == ConnectionPoolSettings.builder().build()
         settings.socketSettings == SocketSettings.builder().build()
-        settings.heartbeatSocketSettings == SocketSettings.builder().readTimeout(20000, TimeUnit.MILLISECONDS).build()
+        settings.heartbeatSocketSettings == SocketSettings.builder().readTimeout(10000, TimeUnit.MILLISECONDS).build()
```

Comment:
Can we add a test that both connect and read timeout of heartbeat socket settings are set to the connect timeout of the socket settings?

Reply:
Done.